src/examples/zkapps/reducer/run-live.ts

```diff
@@ -0,0 +1,143 @@
+import {
+  AccountUpdate,
+  Lightnet,
+  Mina,
+  PrivateKey,
+} from '../../../../dist/node/index.js';
+import { randomAccounts } from '../../utils/randomAccounts.js';
+import { tic, toc } from '../../utils/tic-toc.node.js';
+import { MerkleListReducing } from './actions-as-merkle-list.js';
+
+const useCustomLocalNetwork = process.env.USE_CUSTOM_LOCAL_NETWORK === 'true';
+
+if (!useCustomLocalNetwork) {
+  throw 'Only Lightnet is currently supported';
+}
+
+tic('Run reducer examples against real network.');
+console.log();
+const network = Mina.Network({
+  mina: useCustomLocalNetwork
+    ? 'http://localhost:8080/graphql'
+    : 'https://berkeley.minascan.io/graphql',
+  archive: useCustomLocalNetwork
+    ? 'http://localhost:8282'
+    : 'https://api.minascan.io/archive/berkeley/v1/graphql',
+  lightnetAccountManager: 'http://localhost:8181',
+});
+Mina.setActiveInstance(network);
+
+let { keys, addresses } = randomAccounts('contract', 'user1', 'user2');
+
+let tx, pendingTx: Mina.PendingTransaction, balances, oldBalances;
+
+// compile contracts & wait for fee payer to be funded
+const senderKey = useCustomLocalNetwork
+  ? (await Lightnet.acquireKeyPair()).privateKey
+  : PrivateKey.random();
+const sender = senderKey.toPublicKey();
+
+const sender2Key = useCustomLocalNetwork
+  ? (await Lightnet.acquireKeyPair()).privateKey
+  : PrivateKey.random();
+const sender2 = sender2Key.toPublicKey();
+
+tic('Compiling Merkle List Reducer Smart Contract');
+await MerkleListReducing.compile();
+toc();
+const merkleListReducerContract = new MerkleListReducing(addresses.contract);
+
+let senderSpec = { sender, fee: 1000000000 };
+let sender2Spec = { sender: sender2, fee: 1000000000 };
+
+tic('deploy contracts');
+let deployTx = await Mina.transaction(senderSpec, async () => {
+  AccountUpdate.createSigned(sender).balance.subInPlace(
+    Mina.getNetworkConstants().accountCreationFee
+  );
+  await merkleListReducerContract.deploy();
+});
+pendingTx = await deployTx.sign([senderKey, keys.contract]).send();
+
+await pendingTx.wait();
+toc();
+console.log('working so far');
+// push some actions
+
+tic('dispatch transactions');
+let dispatchTx = await Mina.transaction(senderSpec, async () => {
```

Comment:
In this transactions, one sender creates many actions to reduce in the same block